controllers/auth-controller.js

```diff
@@ -0,0 +1,61 @@
+const { ctrlWrappen } = require("../decorators");
+const { ErrorHttp } = require("../helpers");
+const jwt = require("jsonwebtoken");
+require("dotenv").config();
+const { SECRET_KEY } = process.env;
+const {
+  createNewUsers,
+  getUsersByEmail,
+  checkPasswordUser,
+  createTokenUser,
+  updateUserById,
+  getUserById,
+} = require("../models/users");
+
+const createUserController = async (req, res) => {
+  const { email, password } = req.body;
+  const checkUserByEmail = await getUsersByEmail(email);
+  if (checkUserByEmail) ErrorHttp(409, "users with the email already exists");
```

Comment:
```suggestion
  if (checkUserByEmail) throw ErrorHttp(409, "users with the email already exists");
```
аналогічно в інших місцях